templates/data-sources/cluster_node_pool.md.tmpl

```diff
@@ -1,19 +1,19 @@
 ---
-Title: "Nodepool Resource"
+Title: "Node Pool Resource"
 Description: |-
-    Adding the nodepool resource to a cluster.
+    Fetching the details of node pool resource.
 ---
 
-# Nodepool
+# Node pool
 
-Create, delete, and update nodepools for your clusters using this Terraform module.
+Create, delete, and update node pools for your clusters using this Terraform module.
 
 For clusters that you create in Tanzu Mission Control, you can define a pool of worker nodes on which your workloads can run
 because Tanzu Mission Control cannot provision additional resources in a cluster that is created elsewhere and subsequently attached, you cannot create a node pool in an attached cluster.
 
 ## Example Usage
 
-{{ tffile "examples/resources/tmc_nodepool/resource.tf" }}
+{{ tffile "examples/data-sources/cluster_node_pool/data-source.tf" }}
```

Comment:
resource template is renamed to data-source template?

Reply:
It's the data-source template, should have the data-source example, right?

Reply:
data-source is renamed to resource. So resolving the comment